listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -326,12 +329,24 @@ export default class ListenCard extends React.Component<
             <div className="listen-details">{listenDetails}</div>
           ) : (
             <div className="listen-details">
-              <div title={trackName} className="ellipsis-2-lines">
-                {getTrackLink(listen)}
-              </div>
-              <span className="small text-muted ellipsis" title={artistName}>
+              {trackDuration ? (
+                <div className="title-duration">
+                  <div title={trackName} className="ellipsis-2-lines">
+                    {getTrackLink(listen)}
+                  </div>
+                  <div className="small text-muted" title="Duration">
+                    {isNumber(trackDuration) &&
```

Comment:
How would you feel about adding a dash before the duration as a separator? Superfluous?

<img width="621" alt="image" src="https://user-images.githubusercontent.com/6179856/166911749-f3087acc-cbfe-4a74-9aeb-c19445a8f9ab.png">

If so you would need a non-breaking space between the dash and the number (`&nbsp;` in HTML encoding): 
`-&nbsp;{isNumber(trackDuration) && …`

Reply:
While it sounds like a good idea, adding the dash can make it look out of place like in the first `ListenCard` of your screenshot.
I also think that the separator doesn't fit in the contemporary UI practices 😅

Reply:
I'm OK with that :)

I agree it looks weird with the wrapping text, that's why I put it in the screenshot.

Reply:
I see :)

Then there's nothing else to do here :smile: